### PR TITLE
Fixed confirmation popup bugs

### PR DESCRIPTION
**Pull request by mandyfung**
* Tested on IE11
* Confirmation popup no longer shows up randomly when server side errors happen
* On an unsuccessful submit, confirmation popup will display the error.

### CODE REVIEW

File: scripts/js/confirmPopup.jsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ export default class ConfirmPopup extends React.Component {
   }
 
   submittedMessage(msg) {
+      console.log(msg);
```
Comment (alekhrycaiko) [inline scripts/js/confirmPopup.jsx]: Let's remove this

File: scripts/js/profilerequestform.jsx [hunk 2]
```diff
@@ -22,6 +22,7 @@ export default class ProfileRequestForm extends React.Component {
         super(props);
         autobind(this);
         var popup = null;
+        var alert = null;
```
Comment (alekhrycaiko) [inline scripts/js/profilerequestform.jsx]: is there an advantage to having this as a global over having this as a state property? This doesn't necessarily need to be changed, just kinda wondering.

**Comment by alekhrycaiko** [comment]
Thumbs up ( but please remove the console.log before merging)!

File: scripts/js/profilerequestform.jsx (hunk 2 above)
Comment (mandyfung) [inline scripts/js/profilerequestform.jsx]: Nope, not sure of any advantages.. I just did this to match what I did for the popup

**Pull request merged**